Library/Extensions/Double.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+extension Double {
```

Comment:
Could we rename the file to `Double+Currency.swift` or something to make it a bit clearer that it's an extension and the functionality that it's adding?